### DISCUSSION TITLE
Does it support windows? (without breaking the system)

### DISCUSSION THREAD

**Discussion by ghost** [StevenBlack/hosts#2029] [Q&A] [answered]
I tried to on my windows laptop and it completely bricked my pc.. the DNS service somehow couldn't handle the file and was causing the hole OS to lag and not respond

**Comment by ghost** [comment, marked as answer]
Found the solution, if anyone has the same problem you have to disable the DNS Client Service.

**Reply by rautamiekka** [reply]
That works too, but you should try a **compressed** one designed for M$ in case it doesn't warrant disabling the service, or just to have even more speed: https://scripttiger.github.io/alts/

**Reply by skoenemann** [reply]
I have tried the suggested "compressed" list from Script Tiger. Even with the DNS Client Service disabled, the blacklist is still so large that everything on the laptop runs extremely slow. I have stopped using these blacklists on Windows machines because of this. Everything works fine on Linux, of course, whether or not you use a compressed list.
